Yafc/Windows/ProjectPageSettingsPanel.cs

```diff
@@ -215,7 +205,7 @@ private class ExportMaterial {
                 deflateStream.CopyTo(ms);
                 byte[] bytes = ms.GetBuffer();
                 int index = 0;
-                if (DataUtils.ReadLine(bytes, ref index) != "YAFC" || DataUtils.ReadLine(bytes, ref index) != "ProjectPage") {
```

Comment:
I missed this one, but this is not a valid refactoring.

![afbeelding](https://github.com/have-fun-was-taken/yafc-ce/assets/4461459/f370c6fb-7621-4ad4-a891-a9847effe40d)

This gives a new error in the latest master:
![afbeelding](https://github.com/have-fun-was-taken/yafc-ce/assets/4461459/13b5107e-3495-4c83-afef-17c88c2599de)


On a separate note: Export project page to cliborad exports json, import project page from clipboard expect base64. So i couldnt test this function, but as you see in the watch screenshot it's not working as intended.